pulseapi/settings.py

```diff
@@ -346,6 +348,9 @@ def show_toolbar(request):
     MEDIA_ROOT = root('media/')
     MEDIA_URL = '/media/'
 
+CRM_AWS_SQS_ACCESS_KEY_ID = env('CRM_AWS_SQS_ACCESS_KEY_ID')
+CRM_PETITION_SQS_QUEUE_URL = env('CRM_PETITION_SQS_QUEUE_URL')
```

Comment:
we can probably rename these: `AWS_SQS_ACCESS_KEY_ID` and `SQS_QUEUE_URL`, We also need an `AWS_SQS_SECRET_ACCESS_KEY`